src/interval.jl

```diff
@@ -171,6 +171,22 @@ function Base.last(interval::Interval{T,L,R}) where {T,L,R}
     return R !== Unbounded ? interval.last : nothing
 end
 
+function Base.min(interval::Interval{T,L,R}; precision=nothing) where {T,L,R}
```

Comment:
> Since the other `min` function doesn't set a default precision this function shouldn't either.

Since I added the default of eps(T) to the other function, I assume that it's fine to leave this default?

Reply:
I'd keep the two methods consistent